src/api.js

```diff
@@ -240,6 +240,14 @@ export class LocalStorageApi {
     return this._set('unitSettings', unit)
   }
 
+  getSkipLogoutInfo() {
+    return this._get('skipLogoutInfo')
+  }
+
+  setSkipLogoutInfo(val) {
+    return this._set('skipLogoutInfo', val)
+  }
+
```

Comment:
As we are thinking in store more information related to the session expiration I'd rename it with sessionExpirantionPreferences